package-lock.json

```diff
@@ -2393,6 +2393,11 @@
       "resolved": "https://registry.npmjs.org/boolbase/-/boolbase-1.0.0.tgz",
```

Comment:
Please add the package-lock.json to .gitignore

Reply:
ok